cmd/lakectl/cmd/import.go

```diff
@@ -160,6 +161,12 @@ func newImportProgressBar(visible bool) *progressbar.ProgressBar {
 }
 
 func verifySourceMatchConfiguredStorage(ctx context.Context, client *apigen.ClientWithResponses, source string) {
+	// Adds backwards compatibility for ADLS Gen2 storage import `hint`
+	if strings.Contains(source, "adls.core.windows.net") {
+		Warning("'adls' hint will be deprecated soon, please use the original source url for import")
+		source = strings.Replace(source, "adls.core.windows.net", "blob.core.windows.net", 1)
```

Comment:
I prefer to say what _is_ happening, not only what _isn't_:

```suggestion
		Warning("'adls' hint will be deprecated soon, please use the original source url for import")
		translatedSource = strings.Replace(source, "adls.core.windows.net", "blob.core.windows.net", 1)
		if source != translatedSource {
			Fmt("  Using %s\n", translatedSource)
		}
```